specification/devcenter/DevCenter/Environments/models.tsp

```diff
@@ -0,0 +1,225 @@
+import "@typespec/rest";
+import "@typespec/http";
+import "@azure-tools/typespec-azure-core";
+import "../shared/models.tsp";
+
+using TypeSpec.Versioning;
+using TypeSpec.Rest;
+using TypeSpec.Http;
+
+namespace DevCenterService;
+
+@doc("The type of data a parameter accepts.")
+enum ParameterType {
```

Comment:
The change from enum->union is now supported across emitters so we can change to that format now. Let me know if you need help with this change.